pkgs/by-name/vu/vuetorrent/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildNpmPackage rec {
+  pname = "vuetorrent";
+  version = "2.18.0";
+
+  src = fetchFromGitHub {
+    owner = "VueTorrent";
+    repo = "VueTorrent";
+    rev = "v${version}";
```

Comment:
`refs/tags/v${version}` is a little safer as it avoids ambiguity if there is ever a branch and a tag with the same name, but pointing to different commits